NutritionManagement/src/calculation/NutritionCalculator.java

```diff
@@ -13,25 +11,28 @@ public class NutritionCalculator {
      * @return a new NutritionValue object representing the total Nutrition Value
      */
     public static NutritionValue accumulateNutritionValue(ArrayList<NutritionValue> nutritionValues) {
-        Stream<NutritionValue> nutritions = nutritionValues.stream();
         return new NutritionValue(
-                nutritions.mapToDouble(item -> item.getCalories()).sum(),
-                nutritions.mapToDouble(item -> item.getProtein()).sum(),
-                nutritions.mapToDouble(item -> item.getCarbs()).sum(),
-                nutritions.mapToDouble(item -> item.getFat()).sum());
+                nutritionValues.stream().mapToDouble(item -> item.getCalories()).sum(),
+                nutritionValues.stream().mapToDouble(item -> item.getProtein()).sum(),
+                nutritionValues.stream().mapToDouble(item -> item.getCarbs()).sum(),
+                nutritionValues.stream().mapToDouble(item -> item.getFat()).sum());
     }
 
     /**
      * 
      * @param days
-     * @return a new NutritionValue object representing the average Nutrition Value per day
+     * @return a new NutritionValue object representing the average Nutrition Value
+     *         per day
      */
     public static NutritionValue computeAverage(ArrayList<Day> days) {
-        Stream<NutritionValue> nutritions = days.stream().map(day -> day.getTotalNutrition());
         return new NutritionValue(
-                nutritions.mapToDouble(item -> item.getCalories()).average().orElse(0),
-                nutritions.mapToDouble(item -> item.getProtein()).average().orElse(0),
-                nutritions.mapToDouble(item -> item.getCarbs()).average().orElse(0),
-                nutritions.mapToDouble(item -> item.getFat()).average().orElse(0));
+                days.stream().map(day -> day.getTotalNutrition()).mapToDouble(item -> item.getCalories()).average()
+                        .orElse(0),
```

Comment:
Similar to `accumulateNutritionValue`, improves consistency across methods.

`average().orElse(0)` is a good practice, as it provides a default value in cases where the stream is empty, preventing potential _NoSuchElementException._